cmd/buildah/bud.go

```diff
@@ -70,6 +70,10 @@ func budCmd(c *cli.Context) error {
 	} else {
 		return errors.Errorf("unrecognized image type %q", format)
 	}
+	layers := useLayers()
```

Comment:
Set this in default during option definition, then always just do.

layers = c.Bool("layers")

Reply:
BoolFlag doesn't have a `Value` field. I can set the EnvVar field, which shows which environment variable controls the layers flag.

Reply:
Can you change the description then to add a default "%s", useLayers